### ISSUE TITLE
μs not supported as duration units

### ISSUE THREAD

**Issue by fmoor** [edgedb/edgedb#2194]
It seems like it would make sense to support `μs` as duration units especially since that is how the cli prints it. 

```
edgedb> SELECT <duration>'1us';
{1µs}
edgedb> SELECT <duration>'1μs';
ERROR: InvalidValueError: invalid input syntax for type std::duration: "1μs"
```

- EdgeDB Version: 1.0a8.dev5287+g53ed2609-2020122100~focal.nightly
- OS Version: Linux 5.8.0-40-generic #45~20.04.1-Ubuntu SMP Fri Jan 15 11:35:04 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

**Comment by fmoor** [comment]
Is it better to change the way the CLI prints microsecond durations or to add `replace(val, 'μ', 'u')` to the cast from str to duration? @elprans

**Comment by elprans** [comment]
Definitely.  The CLI is wrong to use non-ASCII characters to represent the values.
